src/R/Sql/Impl/Publish/DacPackageServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using Microsoft.R.Components.Sql.Publish;
+
+namespace Microsoft.VisualStudio.R.Sql.Publish {
+    [Export(typeof(IDacPackageServices))]
```

Comment:
The goal is to be able to load RTVS package without SQL tools and associated assemblies. We now only reference DacFx. We can get away with just `IDacPackageServices`. MEF catalog will see it, but only load for reflection. But - then all code that tries and use the interface will have to check for the SQL tools presence before attempting to use the interface. I think 'GetExportedValueOfDefault` will still throw if export is found, but not loadable - will it? With the provider which always exists call checks for SSDT and then returns null.